husky_components/dropzone/main.js

```diff
@@ -65,7 +67,9 @@ define([], function() {
             keepFilesAfterSuccess: false,
             skin: '',
             dropzoneEnabled: true,
-            cancelUploadOnOverlayClick: false
+            cancelUploadOnOverlayClick: false,
+            maxFilesize: 256, // mb
+            fileTooBigKey: 'husky.upload.error.file-to-big' // can handle {{filesize}}, {{maxFilesize}}, {{filename}
```

Comment:
What does this comment mean? And it looks like there is a '}' missing at the end of the line.